src/common/DisabledButtonInfo.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { ReactNode, useEffect } from 'react'
+import Tooltip, { TooltipProps } from '@mui/material/Tooltip'
+
+function DisabledButtonInfo({
```

Comment:
This sort of wrapper component could be used for any disabled element. Is it worth renaming to reflect that? Something like `DisabledElementTooltip`?

Reply:
This was something I actually considered but never got round - but yeah I agree a name change would be worth doing!